MediaGalleryUi/composer.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "magento/module-media-gallery-ui",
+    "description": "Magento module responsible for the media gallery UI implementation",
+    "require": {
+        "php": "~7.1.3||~7.2.0||~7.3.0",
+        "magento/framework": "*",
+        "magento/module-backend": "*",
+        "magento/module-ui": "*"
+    },
+    "suggest": {
+        "magento/module-cms": "*"
```

Comment:
I think we can suggest installing "magento/module-media-gallery", but not "magento/module-cms".